src/main/java/frc/robot/Alignable.java

```diff
@@ -29,10 +31,4 @@ public double getY() {
     public Rotation2d getAngle(double x, double y) {
```

Comment:
getAngle needs to be moved to the AutoAlignment class as well. The point of this change was to separate the various field elements from the driving, and getAngle(), with its field location calculation, is driving instructions. You will end up having a case statement in there for the various positions, which affords better readability of what is going on.

Reply:
my bad I don't know why I did not copy this method over as well